openpower/package/sb-signing-utils/sb-signing-utils.mk

```diff
@@ -0,0 +1,47 @@
+################################################################################
+#
+#  sb-signing-utils
+#
+################################################################################
+
+SB_SIGNING_UTILS_SITE = ssh://git@github.com/hellerda/sb-signing-utils.git
+SB_SIGNING_UTILS_SITE_METHOD = git
+SB_SIGNING_UTILS_SITE_VERSION = master
+
+SB_SIGNING_UTILS_LICENSE = Apache-2.0
+SB_SIGNING_UTILS_LICENSE_FILES = LICENSE
+SB_SIGNING_UTILS_VERSION ?= master
+
+HOST_SB_SIGNING_UTILS_DEPENDENCIES = host-openssl
+
+define HOST_SB_SIGNING_UTILS_PRE_CONFIGURE_CMDS
+	touch $(@D)/{NEWS,README,AUTHORS,ChangeLog}
+	mkdir -p $(@D)/m4/
+endef
+
+HOST_SB_SIGNING_UTILS_PRE_CONFIGURE_HOOKS += HOST_SB_SIGNING_UTILS_PRE_CONFIGURE_CMDS
+
+HOST_SB_SIGNING_UTILS_AUTORECONF = YES
```

Comment:
Instead of pointing to the git repo, if we point to a release source tarball, this step won't be needed. In fact, having this step makes the build process more fragile.

With a sb-signing-utils tag + release, we can do a "make dist" and upload that tarball to github, and have this makefile fetch that tarball to build from. That way, we don't need to do the reconf.

The "make dist" output of autotools produces a source tarball you should be able to build anywhere, even systems that don't have any autoutils stuff installed ('make dist' includes the configure script itself, rather than just the sources for that).